packages/cli/src/commands/worker.ts

```diff
@@ -64,22 +64,22 @@ export class Worker extends BaseCommand {
 		this.logger.info('Stopping n8n...');
 
 		// Stop accepting new jobs
-		await Worker.jobQueue.pause(true);
+		void Worker.jobQueue.pause(true); // do not block so we can report progress
```

Comment:
can we instead update [`queue.pause` to pass the second argument](https://github.com/n8n-io/n8n/blob/master/packages/cli/src/Queue.ts#L113) (`doNotWaitActive`) as `true` instead of leaving a dangling promise here?